src/solvers/sat/satcheck_cadical.cpp

```diff
@@ -38,12 +38,12 @@ tvt satcheck_cadicalt::l_get(literalt a) const
   return result;
 }
 
-std::string satcheck_cadicalt::solver_text() const
+std::string satcheck_cadical_baset::solver_text() const
 {
   return std::string("CaDiCaL ") + solver->version();
 }
```

Comment:
Shouldn't we override that method to make clear in the output that the solver is used in a non-default configuration?